.plzconfig

```diff
@@ -19,6 +19,7 @@ pleasegotool = //tools/please_go
 embedtool = //tools/please_go_embed
 importpath = github.com/thought-machine/please
 gotool = //third_party/go:toolchain|go
+extldtool = cc
```

Comment:
Why do we need this? Do we need an extld for Please?

Reply:
The go tools will look for gcc on the path anyway. I could put this in .plzconfig.freebsd but I feel this controls that a bit better and allows Please to fail fast rather than waiting for the go tools to fail.

Reply:
Oh ok so we basically always need a C compiler for Go? I guess we inevitably pick up some cgo along the line...

Reply:
> Oh ok so we basically always need a C compiler for Go? I guess we inevitably pick up some cgo along the line...

Oh my apologies, it's for the link static and cgo tests that we need it. Not all go code. If we don't set this, we can still build Please. We just can't run the tests.